src/views/Dashboard.vue

```diff
@@ -22,7 +22,17 @@
             ></b-link>
           </div>
         </b-col>
-        <b-col sm="7" class="d-none d-md-block"> </b-col>
+        <b-col sm="6" class="d-none d-md-block" />
+        <b-col sm="1">
+          <b-form-group id="portfolio-metric-days">
+            <b-form-select
+              id="metric-days-input"
+              v-model="metricDays"
+              class="required"
+              :options="options"
+            />
+          </b-form-group>
```

Comment:
It would be good to have some sort of label here that tells the user what it controls.